Documentation/subcommands/stop.md

```diff
@@ -20,6 +20,14 @@ The `--force` flag will stop a pod forcibly, that is:
 "93e516b0-e84b-40cf-a45b-531b14dfcce2"
 ```
 
+Instead of passing UUID on command line, stop command can read the UUID from a text file.
```

Comment:
I realize this is copied from `rm`, but for consistency with `--force` above it would be nice to have a similar line ("`The `--uuid-file` flag ...").

nit: stop command -> the `stop` command, though this phrasing might change further while including the above.